src/cdk/datetime/tsconfig-build.json

```diff
@@ -2,13 +2,13 @@
   "extends": "../tsconfig-build",
   "files": [
     "public-api.ts",
-    "../typings.d.ts"
+    "../typings.d.ts",
```

Comment:
Is this change intentional?

Reply:
I just wanted to add a comma for future additional files (if necessary)